### PR TITLE
WIP: Remaining Docs

### PR DESCRIPTION
**Pull request by tlyon3**
Translates the following documentation pages to julia:
[x] [Histograms](https://plotly.com/julia/histograms/)
[x] [Error Bars](https://plotly.com/julia/error-bars/)
[ ] [Box Plots](https://plotly.com/julia/box-plots/)
[ ] [Polar Charts](https://plotly.com/julia/polar-charts/)
[ ] [Log Plots](https://plotly.com/julia/log-plots/)
[ ] [Heatmaps](https://plotly.com/julia/heatmaps/)
[ ] [Contour Plots](https://plotly.com/julia/contour-plots/)
[ ] [3D Surface Plots](https://plotly.com/julia/3d-surface-plots/)
[ ] [3D Scatter Plots](https://plotly.com/julia/3d-scatter-plots/)
[ ] [3D Line Plots](https://plotly.com/julia/3d-line-plots/)

### CODE REVIEW

File: julia/error-bars.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+trace = scatter(
+        x=[0, 1, 2],
+        y=[6, 10, 2],
+        mode="markers",
+        error_y=attr(
+            type='percent', # value of error bar given as percentage of y value
+            value=50,
+            visible=true)
+    ))
```
Comment (sglyon) [inline julia/error-bars.md:82]: Extra `)`

File: julia/error-bars.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+trace = scatter(
+        x=[1, 2, 3, 4],
+        y=[2, 1, 3, 4],
+        mode="markers",
+        error_y=attr(
+            type="percent",
+            symmetric=false,
+            value=15,
+            valueminus=25)
+    ))
```
Comment (sglyon) [inline julia/error-bars.md:101]: extra `)`

File: julia/error-bars.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+trace = scatter(
+        x=[1, 2, 3, 4],
+        y=[2, 1, 3, 4],
+        mode="markers",
+        error_x=attr(
+            type="percent",
+            value=15
+        )
+    ))
```
Comment (sglyon) [inline julia/error-bars.md:118]: extra `)

File: julia/error-bars.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+#### Bar Chart with Error Bars
+
+```julia
+using PlotlyJS, CSV, DataFrames
+
+trace1 = bar(
+    name="Control",
+    x=["Trial 1", "Trial 2", "Trial 3"], y=[3, 6, 4],
+    error_y=attr(type="data", array=[1, 0.5, 1.5])
+))
```
Comment (sglyon) [inline julia/error-bars.md:131]: extra `)`

File: julia/error-bars.md [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+trace2 = bar(
+    name="Experimental",
+    x=["Trial 1", "Trial 2", "Trial 3"], y=[4, 7, 3],
+    error_y=attr(type="data", array=[0.5, 1, 2])
+))
```
Comment (sglyon) [inline julia/error-bars.md:136]: extra `)`

**Comment by tlyon3** [comment]
No I will start doing that

**Pull request merged**